### DISCUSSION TITLE
How to view all list of archivements?

### DISCUSSION THREAD

**Discussion by sasaruzic** [community/community#18329] [Other Feature Feedback, Questions, & Ideas]
I am going to see all archivements so that how to get each ones....

**Comment by zynowhere** [comment]
[https://github.com/Schweinepriester/github-profile-achievements](url)

**Reply by RiadZX** [reply]
invalid link

**Comment by TimNekk** [comment]
Maybe this can help https://github.com/Schweinepriester/github-profile-achievements
